mystery-deckbuilder/Assets/Scripts/MainMenu/MainMenu.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+
+public class MainMenu : MonoBehaviour
+{
+    public void Play() 
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex +1);
+    }
+
+    public void Quit()
+    {
+        Debug.Log("Quit");
+        Application.Quit();
+    }
+}
```

Comment:
NIT: Methods still are not commented, but that is okay for something this relatively simple